[docs] [CDCSDK] Updating CDC docs with missing limitations (#12752)
yugabyte/yugabyte-db: docs/content/preview/integrations/cdc/debezium.md

}'
```

{{< note title="Note" >}}

> In case you have a SSL enabled cluster, you need to provide the root certificate to the `database.sslrootcert` configuration property as well.

Reviewer: ```suggestion
If you have an SSL enabled cluster, you need to provide the path to the root certificate in the `database.sslrootcert` configuration property.
```

{{< /note >}}

**In case of SSL enabled YugabyteDB cluster, follow these steps:**

> * Copy the certificate file to your docker container (assuming that the file exists on the root directory of your machine)

Reviewer: Docker is a product name, so we should capitalize it (unless they don't, but I think they do...)
```suggestion
* Copy the certificate file to your Docker container (assuming that the file exists on the root directory of your machine)
```

Author: Yes, they do capitalize it. Thank you for pointing out!

> **In case of SSL enabled YugabyteDB cluster, follow these steps:**
> 
> * Copy the certificate file to your Docker container (assuming that the file exists on the root directory of your machine)
> 
> ```sh
> docker cp ~/root.crt connect:/kafka/
> ```
> 
> * Deploy the connector configuration
> 
> ```sh
> curl -i -X POST -H "Accept:application/json" -H "Content-Type:application/json" \
>   localhost:8083/connectors/ \
>   -d '{
>   "name": "ybconnector",
>   "config": {
>     "connector.class": "io.debezium.connector.yugabytedb.YugabyteDBConnector",
>     "database.hostname":"'$IP'",
>     "database.port":"5433",
>     "database.master.addresses": "'$IP':7100",
>     "database.user": "yugabyte",
>     "database.password": "yugabyte",
>     "database.dbname" : "yugabyte",
>     "database.server.name": "dbserver1",
>     "table.include.list":"public.test",
>     "database.streamid":"d540f5e4890c4d3b812933cbfd703ed3",
>     "snapshot.mode":"never",
>     "database.sslrootcert":"/kafka/root.crt"
>   }
> }'
> ```

Reviewer: Bulleted lists imply that ordering is not significant. This is a procedure, so we need to number the steps.
```suggestion
If you have an SSL enabled YugabyteDB cluster, do the following:

1. Copy the certificate file to your Docker container (assuming that the file exists on the root directory of your machine):

    ```sh
    docker cp ~/root.crt connect:/kafka/
    ```

1. Deploy the connector configuration:

    ```sh
    curl -i -X POST -H "Accept:application/json" -H "Content-Type:application/json" \
      localhost:8083/connectors/ \
      -d '{
      "name": "ybconnector",
      "config": {
        "connector.class": "io.debezium.connector.yugabytedb.YugabyteDBConnector",
        "database.hostname":"'$IP'",
        "database.port":"5433",
        "database.master.addresses": "'$IP':7100",
        "database.user": "yugabyte",
        "database.password": "yugabyte",
        "database.dbname" : "yugabyte",
        "database.server.name": "dbserver1",
        "table.include.list":"public.test",
        "database.streamid":"d540f5e4890c4d3b812933cbfd703ed3",
        "snapshot.mode":"never",
        "database.sslrootcert":"/kafka/root.crt"
      }
    }'
    ```
```
